edm4hep.yaml

```
@@ -6,6 +6,23 @@ options:
   includeSubfolder: True
 
 components:
+    
+    #Vector4f with floats
+    edm4hep::Vector4f:
+      Description: "Generic 4 vector for storing classical 4D coordinates in memory. 4 vector helper functions are in edm4hep::utils"
```

```suggestion
      Description: "Generic vector for storing classical 4D coordinates in memory. Four momentum helper functions are in edm4hep::utils"
```
Maybe we can avoid some of the confusion by calling this a 4D vector and refer to the Lorentz vector as four momentum (and then just try to do that consistently)